frontend/src/utils/amplify-outputs.ts

```diff
@@ -0,0 +1,8 @@
+export const AMPLIFY_OUTPUTS = () => ({
```

Comment:
My intention here was to have a single place for Amplify outputs driven by environment variables.

But that doesn't seem to work with Amplify. Only string literals appear to work.

So this idea might need to be scrapped and we generate our own amplify_ouputs.json file probably via generate-dependencies?